stage_descriptions/sorted-sets-09-sq7.md

```diff
@@ -0,0 +1,78 @@
+In this stage, you'll add support for removing a single zset member using the `ZREM` command.
+
+## The `ZREM` command
+
+The [`ZREM`](https://redis.io/docs/latest/commands/zrem/) command is used to remove a member from a zset given the member's name.
+
+Example Usage:
+
+```bash
+> ZADD racer_scores 8 "Sam-Bodden"
+(integer) 1
+> ZADD racer_scores 10 "Royce"
+(integer) 1
+> ZADD racer_scores 6 "Ford"
+(integer) 1
+> ZADD racer_scores 14 "Prickett"
+(integer) 1
+
+# Remove "Royce" from the zset
+> ZREM racer_scores "Royce"
+(integer) 1
+
+# List the remaining members
+> ZRANGE racer_scores 0 -1
+1) "Ford"
+2) "Sam-Bodden"
+3) "Prickett"
+
+# Remove a non-existing member
+> ZREM racer_scores "non_existing_member"
+(integer)0
+```
+
+It returns the number of members removed from the zset. If the specified member does not exist, 0 is returned.
+
+### Tests
+
+The tester will execute your program like this:
+
+```bash
+$ ./your_program.sh
+```
+
+It will then create a new zset with multiple members.
+
+```bash
+$ redis-cli
+> ZADD zset_key 20.0 member1 (Expecting ":1\r\n")
+> ZADD zset_key 30.1 member2 (Expecting ":1\r\n")
+> ZADD zset_key 100.0 foo (Expecting ":1\r\n")
```

Comment:
We're mixing `foo`/`bar` and `memberN` terminology. Let's avoid this plus try to make it shorter by using a minimal number of elements. If we can just use 2/3 elements to demonstrate what'll happen let's do that.